### PR TITLE
 haproxy entry for cassandra datacenter url

### PR DESCRIPTION
**Pull request by arun-rai**

### CODE REVIEW

File: universe/marathon.json.mustache [hunk 1]
```diff
@@ -48,7 +48,10 @@
   "MARATHON_SINGLE_INSTANCE_APP":"true",
   "DCOS_SERVICE_NAME": "{{service.name}}",
   "DCOS_SERVICE_PORT_INDEX": "0",
-  "DCOS_SERVICE_SCHEME": "http"
+  "DCOS_SERVICE_SCHEME": "http",
+  "HAPROXY_0_HTTP_BACKEND_PROXYPASS_PATH": "/{{service.name}}",
+  "HAPROXY_0_PATH": "/{{service.name}}/v1/datacenter",
+  "HAPROXY_0_VHOST": "{{service.vhost}}"
 },
```
Comment (niteshjain1) [inline universe/marathon.json.mustache]: {{{service.vhost}}}

**Review by niteshjain1 — CHANGES_REQUESTED** [review]

File: universe/marathon.json.mustache (hunk 1 above)
Comment (arun-rai) [reply universe/marathon.json.mustache]: no its only {{service.vhost}}
all the params read like this ex. "DCOS_SERVICE_NAME": "{{service.name}}",

Comment (niteshjain1) [reply universe/marathon.json.mustache]: using {{{}}}, mustache will not escape html chars("&<> etc). Otherwise it escapes html chars.

Comment (arun-rai) [reply universe/marathon.json.mustache]: make scenes, it is getting using like {{#service.external_data_center_urls}} , 
so which one should be used ?? 
{{#}} or {{{}}}  ?

Comment (niteshjain1) [reply universe/marathon.json.mustache]: {{{}}} used this till now in other serivces. Not sure about other syntax

Comment (arun-rai) [reply universe/marathon.json.mustache]: ok changed with working fine with both {{}} and {{{}}}. 

now changed with {{{}}}

**Review by niteshjain1 — APPROVED** [review]

**Pull request merged**